pkg/operator/starter.go

```diff
@@ -61,19 +75,57 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		kubeClient,
 		kubeInformersForNamespaces.InformersFor(operatorNamespace),
 		nil,
+	).WithCSIDriverControllerService(
+		"AWSEFSDriverControllerServiceController",
+		replaceNamespaceFunc(operatorNamespace),
+		"controller.yaml",
+		kubeClient,
+		kubeInformersForNamespaces.InformersFor(operatorNamespace),
+		configInformers,
+		[]factory.Informer{
+			secretInformer.Informer(),
+			nodeInformer.Informer(),
+			infraInformer.Informer(),
+		},
+		csidrivercontrollerservicecontroller.WithSecretHashAnnotationHook(operatorNamespace, secretName, secretInformer),
+		csidrivercontrollerservicecontroller.WithObservedProxyDeploymentHook(),
+		csidrivercontrollerservicecontroller.WithReplicasHook(nodeInformer.Lister()),
```

Comment:
Since we're creating multiple replicas of the controller, any PDB to limit the number of unavailable replicas?